plugin/src/gatsby-node.ts

```diff
@@ -241,12 +241,24 @@ async function sourceChangedNodes(
   if (destroyEvents.length) {
     gatsbyApi.reporter.info(`Removing matching nodes from Gatsby`);
     destroyEvents.forEach((e) => {
-      const node = gatsbyApi
-        .getNodesByType(`Shopify${e.subject_type}`)
-        .find((n) => e.subject_id === parseInt(n.shopifyId as string, 10));
+      /**
+       * When we remove gatsby-node-helpers we'll be able to skip this part,
+       * but for now we'll have to use the same createNodeId function that
+       * this library provides.
+       */
+      const nodeHelpers = createNodeHelpers({
```

Comment:
more of a nit than anything but it might be nice to wrap this in a util to easily build/grab helpers in different places - but it's not super necessary ofc

Reply:
Worth revisiting when we pull the library out of here since I suspect we'll still have some abstractions around node creation.

https://app.clubhouse.io/gatsbyjs/story/24834/remove-gatsby-node-helpers